frontend/components/SetupPage/Create/SetupOnRamp/OnRampPaymentSteps/OnRampPaymentSteps.tsx

```diff
@@ -40,10 +42,19 @@ export const OnRampPaymentSteps = ({
     tokensToBeTransferred,
   );
 
-  const isSetupComplete =
-    isOnRampingStepCompleted &&
-    isSwappingFundsStepCompleted &&
-    isMasterSafeCreatedAndFundsTransferred;
+  // Check if all steps are completed to show the setup complete modal
```

Comment:
yeah, I thought about this logic as well, but it doesn't fix the underlying issue 😓